src/mapml-viewer.js

```diff
@@ -368,6 +372,41 @@ export class MapViewer extends HTMLElement {
               {target: this}}));
       }
     });
+    // pasting layer- and geojson using Ctrl+V 
+    this.parentElement.addEventListener('keydown', function (e) {
+      if(e.keyCode === 86 && e.ctrlKey && document.activeElement.nodeName === "MAPML-VIEWER"){
+        navigator.clipboard
+          .readText()
+          .then(
+            (layer) => {
+              try {
```

Comment:
This code is replicated a few times, here, for dragDrop, and contextmenu paste. I will create a general function for this instead.

Reply:
Fixed [here](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/707/commits/5b2444d5fe69c3ff30adb7a9dc8a4d04026a4c15).